src/components/Datagrid/Datagrid.tsx

```diff
@@ -93,19 +95,17 @@ export interface DatagridProps {
   actionButtonPosition?: "left" | "right";
   recentlyAddedColumn?: string | null; // Enables scroll to recently added column
   onClearRecentlyAddedColumn?: () => void;
+  renderHeader?: (props: DatagridRenderHeaderProps) => ReactNode;
```

Comment:
suggestion: It seems that `renderHeader` is always present so I think we can remove `?`

Reply:
No it's not, because renderHeader exist only in datagrid with variants in product details page